web/sites/default/config/views.view.admin_park_facilities.yml

```diff
@@ -89,23 +89,28 @@ display:
           buttons: false
           action_title: Action
           clear_on_exposed: true
+          force_selection_info: false
           selected_actions:
-            11:
+            8:
               action_id: mark_for_review_action
               preconfiguration:
+                add_confirmation: false
                 label_override: 'Mark for review'
-            12:
+            9:
               action_id: portland_archive_action
               preconfiguration:
+                add_confirmation: false
                 label_override: Unpublish
-            13:
+            11:
               action_id: portland_publish_action
               preconfiguration:
+                add_confirmation: false
                 label_override: Publish
-            17:
-              action_id: pathauto_update_alias
```

Comment:
When I checked Dev/Test, this bulk action was not enabled there. Not sure why this is showing up as a replacement.